cub/cub/util_macro.cuh

```diff
@@ -32,9 +32,11 @@
 
 #pragma once
 
-#include <cuda/std/utility>
+#include <cub/detail/detect_cuda_runtime.cuh>
+#include <cub/util_namespace.cuh>
 
-#include "util_namespace.cuh"
+#include <cuda/std/utility>
+#include <cuda/std/version> // _LIBCUDACXX_HIDDEN, _LIBCUDACXX_{CLANG,GCC}_DIAGNOSTIC_IGNORED
```

Comment:
question: Are `_LIBCUDACXX_HIDDEN` and co part of the public API of libcu++?

Reply:
@ahendriksen no, it's starting with underscore symbol, so it's not part of the public API.